src/crimson/net/Protocol.cc

```diff
@@ -79,7 +79,6 @@ void Protocol::close(bool dispatch_reset,
     return seastar::now();
   }).handle_exception([this] (std::exception_ptr eptr) {
     logger().error("{} ms_handle_reset caught exception: {}", conn, eptr);
-    ceph_abort("unexpected exception from ms_handle_reset()");
```

Comment:
`ceph_abort()` here is on purpose. High-level exceptions should be handled by dispatcher themselves properly and should never leak to the lower-level messenger component.